src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -78,13 +80,28 @@ class OverviewRoute extends React.Component<OverviewRouteProps, State> {
       <>
         <Row>
           <Col sm={colNum}>
-            {showMarketInsights && (
-              <>
-                <MarketInsights artist={artist} />
-                <Spacer mb={1} />
-              </>
-            )}
-
+            <ContextConsumer>
```

Comment:
Up to you, but to avoid this nesting and `ContextConsumer` component, can use a wrapper down below where the fragment container is being declared:

```
export const OverviewRouteFragmentContainer = createFragmentContainer(
  withContext(OverviewRoute),
...)
```

Then, you can just add a `user: User` prop to the props for this component, and access it via `this.props.user`.

Reply:
(`withContext` is https://github.com/artsy/reaction/blob/a7b04e2adb6fad4f4c441e7d2e0e489694dcf849/src/Artsy/SystemContext.tsx#L74 )